README.md

```diff
@@ -13,7 +13,10 @@ This Docker Compose project makes it easy to start and run an Ozone FOSS server
 Install Git, Maven, Docker and Docker compose.
 
 ### Run Ozone FOSS locally
-Create your working directory and as a root user:
+Install Git, Maven and Docker, open a terminal and type in the following:
```

Comment:
I think we can get rid of the above change since it's already covered under the section of `Pre-requisite `